3.Tests/UI/apiappsettings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think it is better to add all of these configurations on the main appsettings.json file, something like this:
```json
{
  "UI": {
    //ui settings
  },
  "API": {
    // api settings
  }
}
```

Reply:
Done.